e/envy/envy_ubuntu_20.04.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Envy
+# Version        : 1.7.1
+# Source repo    : https://github.com/gobuffalo/envy
+# Tested on      : Ubuntu 20.04 + Go latest version (v1.7.1)
+# Script License : GNU Lesser General Public License Version 2.1
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set_env=false
+
+# check installed Golang
+[ -n $(which go) ] && [[ $(go version) > 1.6 ]] && set_env=true
+
+# setup Golang
+if ! $set_env
+then
+
+    cd "$HOME/$USER"
+    mkdir golang go && apt update && apt install wget curl apt-utils git -y --install-recommends
+    [ $? != 0 ] && echo "Installation Failed" && exit 1
+
+    echo "Finding latest version of Go for PPC64LE..."
+    latest=$(curl https://golang.org/VERSION?m=text)
+
+    echo "Downloading latest Go for PPC64LE: ${latest}"
+    $(wget https://dl.google.com/go/${latest}.linux-ppc64le.tar.gz)
+    tar -C golang -xzf "${latest}".linux-ppc64le.tar.gz
+
+    echo "Create the skeleton for your local users go directory"
+    mkdir -p go/{bin,pkg,src}
+
+    echo "Setting up GOPATH"
+    echo "export GOPATH=$(pwd)/go" >> .profile && source .profile
+
+    echo "Setting PATH to include golang binaries"
+    echo "export PATH='$PATH':$(pwd)/golang/go/bin:$GOPATH/bin" >> .profile && source ~/.profile
+
+fi
+
+# install envy package
+go install github.com/gobuffalo/envy@v1.7.1
```

Comment:
Will it run the unit tests ? Are we not passing a command line argument to accept any version and test ?